allensdk/brain_observatory/behavior/behavior_project_cache/tables/sessions_table.py

```diff
@@ -101,6 +106,12 @@ def postprocess_additional(self):
             df=self._df, fetch_api=self._fetch_api
         )
 
+        self._df['behavior_type'] = self._df['session_type'].apply(
+            parse_behavior_context)
+        self._df['image_set'] = self._df['session_type'].apply(
+            parse_stimulus_set)
+        self.df = add_experience_level(self._df)
```

Comment:
There is a typo here self.df. I'm not sure how this got through tests

Reply:
It's because the function edits the data frame in place and then returns it so `self.df` was a copy of the edited `self._df`.